### ISSUE TITLE
Project / App Name ?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/github-backup#13]
`Given` that the App Naming Guidelines: _`"Name should not begin with 'GitHub' or 'Gist'"`_:
![image](https://user-images.githubusercontent.com/194400/36387730-aa744d4e-1591-11e8-85c6-e0ccbab8505a.png)
(_we **cannot** simply call our App "GitHub Issue Backup"_) 

`We need` to come up with a name that:
+ [ ] **describes _exactly_ what** the **App _does_.** (_no "`{{Color || Adjective}} ++ {{Anmial}}` names!!_)
+ [ ] we can get a **Domain Name** for (_regardless of the TLD for now_)
+ [ ] we can get a "**short url**" for. 
+ [ ] we can get **good SEO** for. (_i.e. don't try to use a word or words that are widely used!_)

> e.g: if the Project/App was called **G**it **I**ssue **B**ack**U**p (["GIBU"](https://domainr.com/ibu.com?q=gibu)) we would use a short url like: `gibu.io/dwhq321` would link to  https://github.com/dwyl/hq/issues/321 
(_we would need to define an algorithm for generating the "alpha" part of the `/path` 
in this case I've used the "first two chars of `:owner`" ++ "first two chars of `:repo`"_ ... i.e. `dw+hq`
_where there is a conflict we try more chars from `:owner` or `:repo` until it's **unique**_ ...)

`So that` we can great service for the community!

Naming things is "_hard_" so delegating to our resident "names" person: @iteles 

We need a few _words_ that _describe_ the service we are creating ...
+ Issues
+ Backup
+ Backlog
+ Accountability
+ Log
+ History
+ Version
+ Revision
+ Rollback
+ "Safe"
+ Timetravel 😉 

(_please feel free to add to this list..._)

**Comment by nelsonic** [comment]
How's this for a _laugh_ ... some [domain squatter](https://en.wikipedia.org/wiki/Cybersquatting) wants $4,999 for `gib.io`: 🤣 
![image](https://user-images.githubusercontent.com/194400/36421083-15c87b54-162f-11e8-8d59-a534d2b3009d.png)
I think we'll pass ... 👎 

anyway, we still need a _name_! 💭

**Comment by nelsonic** [comment]
https://domainr.com/gibu.io?q=gibu
![image](https://user-images.githubusercontent.com/194400/37087658-d8c09d84-21f2-11e8-9fae-acf555f42c8b.png)
![image](https://user-images.githubusercontent.com/194400/37087684-f2914b6e-21f2-11e8-9f8e-fae137981559.png)

@iteles can we use this as a "holding domain" / codename until we find a _better_ name...?

**Comment by iteles** [comment]
How would we feel about `gitibu` instead of `gibu`? It's an extra syllable but with `git` in the name I feel like it means more than `gibu`.

<img width="691" alt="screen shot 2018-03-09 at 13 57 22" src="https://user-images.githubusercontent.com/4185328/37210841-fd58873c-23a1-11e8-9b84-f2af789c611b.png">

**Comment by nelsonic** [comment]
@iteles we are trying to get our domain as _short_ as possible. ✂️
I _kinda_ agree that having the _word_ `git` in the name makes it clearerer what it's for.
Could we "_meet in the middle_" with "***gitbu***" ? https://domainr.com/gitbu.com?q=gitbu
![image](https://user-images.githubusercontent.com/194400/37211577-4a14e082-23a4-11e8-90fc-0f0e495b1685.png)
It won't have the extra `i` in it which _stood_ for `issues`.
But that will be _good_ if (_when_) we expand to saving _code_.

**Comment by iteles** [comment]
I very much prefer the more generic 'gitbu' to having the `i` in there anyway.

It pains me that `io`s are the most expensive of all 🙄

**Comment by nelsonic** [comment]
Yeah, `io` _is_ expensive, but at least it's available unlike `gib.io` (_see above_).

Annoyingly this person has the GitHub account: https://github.com/gitbu 😞 
![image](https://user-images.githubusercontent.com/194400/37212748-d4ca6bfe-23a7-11e8-8a2f-8a7bb8aedeff.png)
So we _may_ need to use something _else_ as the GitHub username ... 🤔

**Comment by iteles** [comment]
https://github.com/gitbuio ?

**Comment by nelsonic** [comment]
@iteles yeah, that's a good "stop gap" (alternative). 👍

**Comment by iteles** [comment]
@nelsonic Buy this for the redirect to whatever we choose?
<img width="845" alt="screen shot 2018-03-09 at 14 50 07" src="https://user-images.githubusercontent.com/4185328/37213320-99bba828-23a9-11e8-81cb-8b40cd501ee5.png">

**Comment by nelsonic** [comment]
yeah, that could be good for re-directing and/or as use for landing page.

**Comment by Cleop** [comment]
I ❤️ gitbu

**Comment by nelsonic** [comment]
@iteles what are your thoughts on using https://github.com/gitbuco instead of `.io`
we get the same _length_ and the domain is available but it _sounds_ more like a "biscuit company" 
and doesn't have the "Chagossians" controversy ...
https://gigaom.com/2014/06/30/the-dark-side-of-io-how-the-u-k-is-making-web-domain-profits-from-a-shady-cold-war-land-deal/

**Comment by iteles** [comment]
Erm, got a little ahead of myself.

<img width="843" alt="screen shot 2018-03-09 at 14 29 51" src="https://user-images.githubusercontent.com/4185328/37212753-da0d5234-23a7-11e8-9264-79e1848e74de.png">

I can still get `.co`?
<img width="855" alt="screen shot 2018-03-09 at 15 20 44" src="https://user-images.githubusercontent.com/4185328/37215171-217f4b3e-23af-11e8-854f-1fa288cc0291.png">

**Comment by nelsonic** [comment]
@iteles did you end up registering gitbu.co? (_it still appears to be "available"..._)
![image](https://user-images.githubusercontent.com/194400/37253345-800da2ca-2528-11e8-98e5-0bdd1bbac643.png)

**Comment by nelsonic** [comment]
Registered https://github.com/gitbuco GitHub User (_let us know if you need the password ..._)

**Comment by iteles** [comment]
@nelsonic, I did register `.co` :blush:

Please send @cleop the login details for the user.

**Comment by nelsonic** [comment]
https://www.namecheap.com/domains/registration/results/?domain=gitbu
![image](https://user-images.githubusercontent.com/194400/194246896-a8386423-f82c-47f1-b777-65c235883429.png)
